source/Calamari.Tests/KubernetesFixtures/InstallTools.cs

```diff
@@ -217,6 +218,45 @@ public async Task InstallGCloud()
                         return GetGcloudExecutablePath(destinationDirectoryName);
                     });
             }
+            
+            if (CalamariEnvironment.IsRunningOnWindows)
+            {
+                InstallGkeAuthPlugin();
+            }
+        }
+        
+        void InstallGkeAuthPlugin()
+        {
+            var variables = new Dictionary<string, string>();
+            var pythonCopyPath = ExecuteCommandAndReturnResult(GcloudExecutable, "components copy-bundled-python", ".", variables);
+            variables.Add("CLOUDSDK_PYTHON", pythonCopyPath.commandOutput);
+
+            var gkeComponent = ExecuteCommandAndReturnResult($"\"{GcloudExecutable}\"", "components list --filter=\"Name=gke-gcloud-auth-plugin\" --format=\"json\"", ".", variables);
+            var gkeComponentJObject = JArray.Parse(gkeComponent.commandOutput).First();
+            var installedState = gkeComponentJObject["state"]["name"].Value<string>();
+            if (installedState != "Installed")
+            {
+                ExecuteCommandAndReturnResult(GcloudExecutable, "components install gke-gcloud-auth-plugin --quiet", ".", variables);
+            }
+        }
+
+        (int ExitCode, string commandOutput) ExecuteCommandAndReturnResult(string executable, string arguments, string workingDirectory, Dictionary<string, string> variables)
+        {
+            var stdOut = new StringBuilder();
+            var stdError = new StringBuilder();
+            var commandExitCode = SilentProcessRunner.ExecuteCommand(executable,
+                                                                     arguments,
+                                                                     workingDirectory,
+                                                                     variables,
+                                                                     (Action<string>)(s => stdOut.AppendLine(s)),
+                                                                     (Action<string>)(s => stdError.AppendLine(s))).ExitCode;
+
+            if (commandExitCode != 0)
+            {
+                throw new Exception($"{stdOut}{stdError}");
```

Comment:
We could add more structure to the message, for example, put stdOut and stdError on separate lines and give a heading to each. Simply concatenating them may not give an easy-to-read message.